src/Microsoft.ML.Core/Data/MetadataUtils.cs

```diff
@@ -295,7 +295,7 @@ public static IEnumerable<int> GetColumnSet(this ISchema schema, string metadata
         /// Returns <c>true</c> if the specified column:
         ///  * is a vector of length N (including 0)
         ///  * has a SlotNames metadata
-        ///  * metadata type is VBuffer&lt;DvText&gt; of length N
+        ///  * metadata type is VBuffer&lt;ReadOnlyMemory&gt; of length N
```

Comment:
>ReadOnlyMemory [](start = 43, length = 14)

If you wanted to read it out it would be not `ReadOnlyMemory` but `ReadOnlyMemory&ltchar&gt`, but this is why we have `<see` tags in the XML docs.